web/util/date.ts

```diff
@@ -30,6 +26,22 @@ export const formatTimeDuration = (start: Date, duration: number) => {
   return `PT${hours ? hours + 'H' : ''}${minutes ? minutes + 'M' : ''}`;
 };
 
+const differingMeridiem = (d1: Date, d2: Date) =>
+  d1.getHours() >= 12 !== d2.getHours() >= 12;
+
+export const formatTimeRange = (
+  start: Date,
+  duration: number,
+  timezone: string
+) => {
+  const end = addMinutes(start, duration);
+  return `${formatTime(
+    start,
+    timezone,
+    differingMeridiem(start, end)
+  )}–${formatTime(end, timezone)}`;
```

Comment:
Minor thing, but per Carrie's comment there should be spaces around the en dash iff `differingMeridiem(start, end)` is true. It does perhaps look slightly confusing without, as if the "AM" "belongs" with the end time:

> 9:00 AM–1:00 PM

vs

> 9:00 AM – 1:00 PM

Reply:
Good eye! :eyes: I'd missed this difference. 5cbdcb1